>From 609b6b4899953cce84dc72a7ec0a81ccfc9880b0 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Mon, 26 Mar 2012 22:11:05 +0200
Subject: [PATCH] Fix a typo in debhelper.pod
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Closes: #665891
---
 debhelper.pod |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/debhelper.pod b/debhelper.pod
index 383172e..af912a7 100644
--- a/debhelper.pod
+++ b/debhelper.pod
@@ -87,7 +87,7 @@ characters (B<?> and B<*> and B<[>I<..>B<]> character classes) in the files.
 You can also put comments in these files; lines beginning with B<#> are
 ignored.
 
-The syntax of these files is intentionally kept very simple to make then
+The syntax of these files is intentionally kept very simple to make them
 easy to read, understand, and modify. If you prefer power and complexity,
 you can make the file executable, and write a program that outputs
 whatever content is appropriate for a given situation. When you do so,
-- 
1.7.9.1

